### PR TITLE
Docker compose file version update and other minor tuning

### PR DESCRIPTION
**Pull request by paazmaya**
Trying to make sense what is happening within these compose files and how other files are related to them.

Where does `VRT_VERSION` come from?

### CODE REVIEW

**Comment by paazmaya** [comment]
@pashidlos how should it be named? I changed the origin name to the ui version, but the target name kept the same...

File: docker-compose.elastic.logging.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-version: "2.1"
+version: "3.8" # Docker Engine minimum version 19.03.0
```
Comment (pashidlos) [inline docker-compose.elastic.logging.yml:1]: could you check if it's working?
had an issue with that https://github.com/Visual-Regression-Tracker/Visual-Regression-Tracker/issues/163

**Review by pashidlos — APPROVED** [review]
LGTM if docker version is not and issue anymore

File: docker-compose.elastic.logging.yml (hunk 1 above)
Comment (paazmaya) [reply docker-compose.elastic.logging.yml:1]: Yes, I have been using this branch with
* Docker Compose version v2.12.2
* Docker version 20.10.21

**Pull request merged**